src/main/java/com/savi/geo/timezones/TzWhere.java

```diff
@@ -0,0 +1,61 @@
+//?? TODO:  REVIEW:  How much flexibility should we support right now?
+// - Mainly, do/will we need to provide the option to, instead of using just the
+//   default shapefile (classpath resource files in the Jar file), use another
+//   shapefile(s) at the same time (in the same run and without classloader
+//   trickiness)?  (Relates to how to structure code now (re interface for
+//   internal IndexImpl, re static vs. non-static methods).)
+//
+// - Should we currently provide the option to override the default shapefile
+//   with one static replacement (probably by defining system property used to
+//   override default classpath resource pathname to shapefile files)?
+
+//?? TODO:  REVIEW:  Should proper-noun--based class name "TzWhere" be changed
```

Comment:
The more I think about it, the more I'm ok using TzWhere. If people have used the Python or Javascript versions, they know what they're getting here.